model/src/main/scala/cromwell/pipeline/model/wrapper/RunId.scala

```diff
@@ -21,4 +21,5 @@ object RunId extends Wrapped.Companion {
   )
 
   def random: RunId = new RunId(java.util.UUID.randomUUID().toString)
+  def convertRun(value: String): RunId = new RunId(value)
```

Comment:
The whole point of wrappers is that they won't allow someone to create object in illegal state, e.g. `RunId("definitely not UUID")`. This method bypasses these checks. You should remove it and use `RunId.from()` instead.

Reply:
This is good to know. I was not sure how to get the `RunId` object that was the reason i created the method. Thanks for pointing me to use the correct syntax, i have applied `from` method